alphagov/paas-admin
# Move to npm v7 and npm audit fix

What
----

Move to npm v7 and ran npm audit fix

npm v7 introduces a new version of the package-lock file

```
# npm audit report

ini  <1.3.6
Prototype Pollution - https://npmjs.com/advisories/1589
fix available via `npm audit fix`
node_modules/fsevents/node_modules/ini

minimist  <0.2.1 || >=1.0.0 <1.2.3
Prototype Pollution - https://npmjs.com/advisories/1179
fix available via `npm audit fix --force`
Will install sass-lint@1.13.0, which is a breaking change
node_modules/fsevents/node_modules/minimist
node_modules/fsevents/node_modules/rc/node_modules/minimist
node_modules/minimist
  gonzales-pe-sl  *
  Depends on vulnerable versions of minimist
  node_modules/gonzales-pe-sl
    sass-lint  1.6.0 - 1.7.0 || 1.12.1 || >=1.13.1
    Depends on vulnerable versions of gonzales-pe-sl
    node_modules/sass-lint
  mkdirp  0.4.1 - 0.5.1
  Depends on vulnerable versions of minimist
  node_modules/fsevents/node_modules/mkdirp

5 low severity vulnerabilities
```

How to review
-------------

- update npm to v7 with `npm install npm -g`
- update version of node with say nvm,  `nvm install lts/Erbium`
- install latest project dependencies with `npm install`
- run locally or deploy to own env
- all should be working

Who can review
---------------

not @kr8n3r
